app/src/main/java/com/example/soptseminar1/HomeActivity.kt

```diff
@@ -2,10 +2,51 @@ package com.example.soptseminar1
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import com.example.soptseminar1.databinding.ActivityHomeBinding
 
 class HomeActivity : AppCompatActivity() {
+
+    private var position = FOLLOWER
+    private lateinit var binding: ActivityHomeBinding
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_home)
+        binding = ActivityHomeBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        initTransactionEvent()
+    }
+
+    private fun initTransactionEvent(){
+        val fragment1 = FollowerFragment()
+        val fragment2 = RepositoryFragment()
+
+        supportFragmentManager.beginTransaction().add(R.id.main_fragment, fragment1).commit()
```

Comment:
```suggestion
        supportFragmentManager.beginTransaction()
            .add(R.id.main_fragment, fragment1)
            .commit()
```
이렇게 코드를 작성하면, `beginTransaction()`을 실행하고 `add()`를 실행한 후 `commit()`을 실행하는구나 ~ 가 눈에 더 잘 들어오지 않을까요? 이렇게 자동으로 포맷을 맞춰주는 기능은 Android Studio 자체적으로 단축키로 제공합니다. `CTRL+ALT+L` 이라고 하네요.